app/policies/document_policy.rb

```diff
@@ -14,4 +14,28 @@ def publish?
   end
 
   alias_method :unpublish?, :publish?
+
+  def pre_production_formats
+    Rails.env.development? ? [] : PRE_PRODUCTION
+  end
+
+  def removed_pre_production?
+    !pre_production_formats.include?(document_class.document_type)
+  end
```

Comment:
I don't understand this method name.

Is it asking whether the current document type should be hidden if this isn't a production environment?